pkg/api/steve/proxy/proxy.go

```diff
@@ -65,7 +65,7 @@ func NewProxyMiddleware(sar v1.AuthorizationV1Interface,
 
 	mux := gmux.NewRouter()
 	mux.UseEncodedPath()
-	mux.PathPrefix("/api").HandlerFunc(proxyHandler.authLocalCluster(mux))
+	mux.PathPrefix("/api").HandlerFunc(proxyHandler.authLocalCluster(localCluster))
```

Comment:
I don't understand this well enough to say whether this is good or bad, but seems noteworthy ?